app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -45,6 +45,14 @@ def update_attributes!(*args)
         dynflow_task_wrap(:save) { super(*args) }
       end
 
+      # this can be used when HostActionSubject is used for orchestration but you want to avoid
+      # triggering more tasks by ActiveRecord callbacks within run/finalize phase of your task
+      def save_with_disabled_dynflow_hooks
+        @_disable_dynflow_hooks = true
+        save
```

Comment:
what about just calling `save_without_dynflow_task_wrap` from here. if you have reasons not to, you should save the original value `@_disable_dynflow_hooks` and restore it in `ensure` block